content/enterprise_influxdb/v1.8/administration/anti-entropy/_index.md

```diff
@@ -287,15 +287,17 @@ If a data node suddenly disappears due to a catastrophic hardware failure or for
 
 _View the [Replacing Data Nodes](/enterprise_influxdb/v1.8/guides/replacing-nodes/#replace-data-nodes-in-an-influxdb-enterprise-cluster) documentation for instructions on replacing data nodes in your InfluxDB Enterprise cluster._
 
-### Replacing a machine that is running a data node
+### Replacing a machine running a data node
 
-Perhaps you are replacing a machine that is being decommissioned, upgrading hardware, or something else entirely.
-The Anti-Entropy service will automatically copy shards to the new machines.
+To replace a machine running a data node, doing one of the following:
 
-Once you have successfully run the `influxd-ctl update-data` command, you are free
-to shut down the retired node without causing any interruption to the cluster.
-The Anti-Entropy process will continue copying the appropriate shards from the
-remaining replicas in the cluster.
+- To start the Anti-Entropy service and copy shards to the new machine, on a meta node in your cluster, run [`influxd-ctl update-data`](/enterprise_influxdb/v1.8/administration/cluster-commands/#update-data), and then shut down the retired data node without causing any interruption to the cluster. The Anti-Entropy process continues to copy the appropriate shards from the remaining replicas in the cluster.
+- For nodes where the data set is too large or shaped in such a way that AE cannot be enabled, complete the following steps:
```

Comment:
Thank you, @dgnorton! Re: "too large" and "shaped" in a way the AE cannot be enabled....we typically try to omit terms that are open to interpretation. Are there any specifics that you're comfortable committing to that we can add here? And/or is there any indicator/way to determine that AE cannot be enabled? Do folks try to turn AE on and just doesn't start?

Reply:
@kelseiv "too large" depends on the shape of their data and computing resources available so there isn't really an exact guideline. They can try enabling AE and if CPU usage spikes too high for the cluster to operate then disable it. If it spikes but the cluster continues to run, it can be left on and see if CPU usage settles back to an acceptable level after a day. There will be a period of time after AE is first enabled where it's generating shard digests and comparing them with other nodes so some increase in CPU and disk I/O is expected.